### ISSUE TITLE
`else, new line, if` is interpreted as else if but this it not (always) intended

### ISSUE THREAD

**Issue by michaellilltokiwa** [tokiwa-software/fuzion#329]
This example produces the error: `Inconsistent indentation`.
```
ex =>
  some_fun bool is
    if true
      true
    else
      if true
        say "hello"
      else
        say "hello"
      true
```
We could either improve the error message or change syntax to require `else if` to be on the same line.
My preference leans towards the later option.

**Comment by fridis** [comment]
Good examle. I think this is related to #268: A solution might be to have a block that does not start a new line (like the `if` in `else if`) set the indentation reference to the first non-space character of that line, i.e., in
```
  else if ...
```
for the `if` we would use the first `e` as reference, while for
```
  else
    if ...
```
that would be the `i`.
Would be cool if that would solve both this issue and #268.

**Comment by michaellilltokiwa** [comment]
related #266

**Comment by fridis** [comment]
fixed by #1297
